make/cats

```diff
@@ -38,7 +38,20 @@ kubectl create --namespace="${NAMESPACE}" --filename="kube/bosh-task/acceptance-
 
 stampy "${GIT_ROOT}/scf_metrics.csv" "$0" make-cats::create end
 
-sleep 5 ; # Slap dash: Wait for container to be up and running
+i=0
+
+set +o errexit
+while [ "$(k get pod cf:^acceptance-tests -o=json | jq -r .status.phase)" != "Running" ]
```

Comment:
Maybe just make note of the possible trouble for now.
I mean, :tophat: is currently not working anymore either, and might not be resurrected, making it moot.